components/ArtistSetting/StandaloneYoutubeComponent/ChannelInfo.tsx

```diff
@@ -0,0 +1,72 @@
+import { Tooltip } from "@/components/common/Tooltip";
+import { cn } from "@/lib/utils";
+import formatFollowerCount from "@/lib/utils/formatFollowerCount";
+import { useQuery } from "@tanstack/react-query";
+import { Youtube } from "lucide-react";
+
+const ChannelInfo = ({ dense, artistAccountId }: { dense?: boolean; artistAccountId: string }) => {
+  const { data, isLoading } = useQuery({
+    queryKey: ["youtube-channel-info", artistAccountId],
+    queryFn: () =>
+      fetch(
+        `/api/youtube/channel-info?artist_account_id=${artistAccountId}`
```

Comment:
Do we need all of these?
- /api/youtube/token
- /api/youtube/status
- /api/youtube/channel
IMO, get rid of all of these and keep only `/api/youtube/channel`

principle
```
less is more
simple is best
small is beautiful
```